requirements.txt

```diff
@@ -3,9 +3,10 @@
 # activated.
 # For more information on how to set up conda with bioconda channel see:
 # http://bioconda.github.io/#install-conda
-# This file can be installed with "conda install --file requirements-test.txt".
+# This file can be installed with "conda install --file requirements.txt".
 
-cromwell
+cromwell>=53
```

Comment:
How come we can't use cromwell 51 for instance?

Reply:
Some construct I used before to merge the CCS scatters failed with crowell 51. The latest version works with cromwell 51 again